tripal_chado/includes/TripalImporter/GFF3Importer.inc

```diff
@@ -1531,9 +1541,10 @@ class GFF3Importer extends TripalImporter {
     }
 
     // Cache the GFF feature details for later lookup.
-    if (strcmp($gff_feature['uniquename'], $gff_feature['landmark']) != 0) {
+    // SOFIA: THIS IS IT, HERE is where landmarks dont get cached.
+#    if (strcmp($gff_feature['uniquename'], $gff_feature['landmark']) != 0) {
       $this->cacheFeature($gff_feature);
-    }
+#    }
```

Comment:
Can you remove the inline comments  in the lines above?

Reply:
Done.